### PR TITLE
feat: add csharp sdk to dockerfile

### PR DESCRIPTION
**Pull request by hpstory**
If this PR is related to coding or code translation, please fill out the checklist and paste the console outputs to the PR.

- [x] I've tested the code and ensured the outputs are the same as the outputs of reference code.
- [x] I've checked the code (formatting, comments, indentation, file header, etc) carefully.
- [x] The code does not rely on a particular environment or IDE and can be executed on a standard system (Win, macOS, Ubuntu).

### CODE REVIEW

**Comment by hpstory** [comment]
Use docker-compose --env-file .\codes\.env -f .\codes\docker-compose.yml up

File: codes/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+LANGS_SDK="python cpp java csharp"
```
Comment (krahets) [inline codes/.env:1]: The languages to be installed are selected using `LANGS` arg in Dockerfile. Is this duplicated?

File: codes/docker-compose.yml [hunk 2]
```diff
@@ -1,7 +1,10 @@
 version: '3.8'
 services:
   hello-algo-code:
-    build: .
+    build:
+      context: .
+      args:
+        LANGS: ${LANGS_SDK}
```
Comment (krahets) [inline codes/docker-compose.yml:7]: Same as above.

File: codes/.env (hunk 1 above)
Comment (hpstory) [reply codes/.env:1]: Actually not, the LANGS in the dockerfile entirely depend on the variables in the docker-compose file. I think we need a default value here, but the dockerfile still needs to update.

File: codes/docker-compose.yml [hunk 3]
```diff
@@ -1,7 +1,12 @@
 version: '3.8'
 services:
   hello-algo-code:
-    build: .
+    build:
+      context: .
+      args:
+        # 设置需要安装的语言，使用空格隔开
+        # Set the languages to be installed, separated by spaces
+        LANGS: "python cpp csharp"
```
Comment (krahets) [inline codes/docker-compose.yml:9]: I prefer setting the LANGS here with out adding a .ENV file. How do you think of it?

File: codes/.env (hunk 1 above)
Comment (krahets) [reply codes/.env:1]: OK, I removed the default values of `LANGS` in Dockerfile to make it fully controlled by `docker-compose`.

File: codes/docker-compose.yml (hunk 3 above)
Comment (hpstory) [reply codes/docker-compose.yml:9]: remove env file

**Review by krahets — APPROVED** [review]
Thanks for the PR!

**Pull request merged**